env.sh.example

```diff
@@ -21,7 +21,10 @@ export VERTEX_LOCATION=europe-west2
 export VERTEX_NETWORK= # optional
 export VERTEX_PROJECT_ID=my-gcp-project
 
+# Suffix (e.g. _<your name>) to facilitate running concurrent pipelines in the same Google Cloud project. Populate if working in a team to avoid overwriting resources during development 
```

Comment:
```suggestion
# Suffix (e.g. '_<your name>') to facilitate running concurrent pipelines in the same Google Cloud project. Populate if working in a team to avoid overwriting resources during development 
```

Reply:
Don't think the single quotes are necessary? The rest of the env vars are defined without any quotes

Reply:
I think Felix is just suggesting quotes for the purposes of the clarity in the docs, not because they need to be specified with quotes in `env.sh` - @felix-datatonic ?